docker/main/docker-deploy.yml

```diff
@@ -79,10 +79,16 @@ services:
       - DOCKER_SECRET_REDIS_PASS=myredis_pass
```

Comment:
Comment: It might be worth adding all of these into a .env file or some sort of configuration setup due to the number of variables. Looks like this is going to scale at 3 variables per service? If that's the case a more centralized setup might be helpful in the future (especially since it's all "host", "port", "SSL Directory")